measures/BuildExistingModel/measure.rb

```diff
@@ -367,17 +371,39 @@ def run(model, runner, user_arguments)
     measures['HPXMLtoOpenStudio'][0]['debug'] = args['debug'].get if args['debug'].is_initialized
     measures['HPXMLtoOpenStudio'][0]['add_component_loads'] = args['add_component_loads'].get if args['add_component_loads'].is_initialized
 
-    if not apply_measures(hpxml_measures_dir, { 'BuildResidentialHPXML' => measures['BuildResidentialHPXML'], 'BuildResidentialScheduleFile' => measures['BuildResidentialScheduleFile'], 'HPXMLtoOpenStudio' => measures['HPXMLtoOpenStudio'] }, new_runner, model, true, 'OpenStudio::Measure::ModelMeasure', 'existing.osw')
-      new_runner.result.warnings.each do |warning|
-        runner.registerWarning(warning.logMessage)
-      end
-      new_runner.result.info.each do |info|
-        runner.registerInfo(info.logMessage)
+    # Specify measures to run
+    if run_hescore_workflow
+      measures['BuildResidentialHPXML'][0]['apply_defaults'] = true
+      measures_hash = { 'BuildResidentialHPXML' => measures['BuildResidentialHPXML'] }
+    else
+      measures_hash = { 'BuildResidentialHPXML' => measures['BuildResidentialHPXML'], 'BuildResidentialScheduleFile' => measures['BuildResidentialScheduleFile'], 'HPXMLtoOpenStudio' => measures['HPXMLtoOpenStudio'] }
+    end
+
+    if not apply_measures(hpxml_measures_dir, measures_hash, new_runner, model, true, 'OpenStudio::Measure::ModelMeasure', 'existing.osw')
+      register_logs(runner, new_runner)
+      return false
+    end
+
+    # Run HEScore Measures
+    if run_hescore_workflow
+      measures['HPXMLtoHEScore'] = [{ 'hpxml_path' => hpxml_path, 'output_path' => File.expand_path('../hes.json') }]
```

Comment:
I'd store `File.expand_path('../hes.json')` in a variable in case it gets changed in the future...